### PR TITLE
docs: Fixing links

### PR DESCRIPTION
**Pull request by mitar**
## Proposed changes

Just small fixes to paths.

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
- [x] I have read the [security policy](../security/policy)
- [x] I confirm that this pull request does not address a security vulnerability. If this pull request addresses a security
vulnerability, I confirm that I got green light (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation within the code base (if appropriate)

### CODE REVIEW

**Comment by aeneasr** [comment]
These links are correct and working

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -24,8 +24,8 @@ Put an `x` in the boxes that apply. You can also fill these out after creating t
 them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
 -->
 
-- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
-- [ ] I have read the [security policy](../security/policy)
```
Comment (mitar) [inline .github/PULL_REQUEST_TEMPLATE.md:28]: Where does `../security/policy` point to?

Comment (aeneasr) [reply .github/PULL_REQUEST_TEMPLATE.md:28]: https://github.com/ory/fosite/network/alerts

Just click the link in your PR ;)

Comment (mitar) [reply .github/PULL_REQUEST_TEMPLATE.md:28]: Oh, I was clicking on it while browsing the repository. OK.

**Pull request closed without merging**